lib/mayaUsd/commands/baseListShadingModesCommand.cpp

```diff
@@ -75,41 +82,59 @@ MayaUSDListShadingModesCommand::doIt(const MArgList& args) {
     }
 
     if (argData.isFlagSet("export")) {
-        // We have these default exporters which are always 1-2-3 in the options:
-        appendToResult(UsdMayaShadingModeRegistry::GetExportConversionInfo(
+        // We have these default exporters which are always 1-2 in the options:
+        appendToResult(UsdMayaShadingModeRegistry::GetMaterialConversionInfo(
                            UsdImagingTokens->UsdPreviewSurface)
                            .niceName.GetText());
-        appendToResult(
-            UsdMayaShadingModeRegistry::GetExporterNiceName(UsdMayaShadingModeTokens->displayColor)
-                .c_str());
```

Comment:
displayColor is no longer a shadingMode export option. We still allow it via the exportDisplayColor geom export option.